src/main/resources/db/migration/entries/V1000__CLIENT.sql

```diff
@@ -1,5 +0,0 @@
-INSERT INTO address(id, street, postcode, city, country)
```

Comment:
I prefer that we keep the fake date in the entries. Like this way, in DEV we have some data to play with the app.

Reply:
Can we add this later on? I will add some as soon as will start writing tests & as soon we will have table structure set